src/main/java/au/com/dmg/fusion/response/storevalueresponse/StoredValueResult.java

```diff
@@ -0,0 +1,41 @@
+package au.com.dmg.fusion.response.storevalueresponse;
+
+import au.com.dmg.fusion.data.StoredValueInformation;
+import com.squareup.moshi.Json;
+
+
+public class StoredValueResult extends StoredValueInformation {
+    @Json(name = "StoredValueAccountStatus")
+    private final StoredValueAccountStatus storedValueAccountStatus;
+
+    public StoredValueAccountStatus getStoredValueAccountStatus(){ return storedValueAccountStatus; }
+
+    public static class Builder extends StoredValueInformation.Builder {
+        private StoredValueAccountStatus storedValueAccountStatus;
+
+        public Builder() {
+        }
+
+        Builder(StoredValueAccountStatus storedValueAccountStatus){
+            this.storedValueAccountStatus = storedValueAccountStatus;
+        }
+
+        public Builder storedValueAccountStatus(StoredValueAccountStatus storedValueAccountStatus){
+            this.storedValueAccountStatus = storedValueAccountStatus;
+            return Builder.this;
+        }
+
+        public StoredValueResult build() {
+            if (this.storedValueTransactionType == null) {
```

Comment:
IdentificationType and StoredValueID are also required.

Reply:
Hi @i-grace , the identificationType and storedValueID are required under StoredValueAccountStatus. I updated the test cases instead to capture this.

Reply:
There's currently validation in StoredValueAccountStatus for those fields?

public StoredValueAccountStatus build(){
            return new StoredValueAccountStatus(this);
        }

Reply:
Sorry, it's under StoredValueAccountID @i-grace

Reply:
Hi @i-grace -- I just added the null checking on the StoredValueAccountStatus build.